account_bank_statement_import_save_file/models/account_bank_statement_import.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import base64
+import inspect
+from openerp import models, api
+
+
+class AccountBankStatementImport(models.TransientModel):
+    _inherit = 'account.bank.statement.import'
+
+    @api.model
+    def _import_statement(self, statement):
+        (statement_id, notifications) = \
+            super(AccountBankStatementImport, self)._import_statement(
+                statement)
+        if statement_id:
+            # get raw file data from the stack
+            def get_data_file(frame):
```

Comment:
back then, no. Now, it makes more sense to overwrite https://github.com/OCA/bank-statement-import/blob/8.0/account_bank_statement_import/account_bank_statement_import.py#L71

I'll put this to work in progress until i can follow up